src/goto-symex/symex_target_equation.h

```diff
@@ -29,6 +29,8 @@ class decision_proceduret;
 class namespacet;
 class prop_convt;
 
+/// Inheriting the interface of \c symex_targett this class represents the SSA
+/// form as a list of \c SSA_stept.
```

Comment:
:pick: Represents the SSA form of what? How does it extend the base class?

Reply:
Done.